x/exchange/types/params.go

```diff
@@ -21,11 +22,11 @@ var (
 )
 
 var (
-	DefaultFees = Fees{
-		MarketCreationFee:          sdk.NewCoins(sdk.NewInt64Coin(sdk.DefaultBondDenom, 1000000)),
-		DefaultMakerFeeRate:        sdk.NewDecWithPrec(-15, 4), // -0.15%
-		DefaultTakerFeeRate:        sdk.NewDecWithPrec(3, 3),   // 0.3%
-		DefaultOrderSourceFeeRatio: sdk.NewDecWithPrec(5, 1),   // 50%
+	DefaultMarketCreationFee = sdk.NewCoins(sdk.NewInt64Coin(sdk.DefaultBondDenom, 1000000))
```

Comment:
Currently, v5 UpgradeHandler uses DefaultParams, As a result, there are some cases where 1DefaultBondDenom1 is used and goes into a `stake`, not `ucre`, and it seems that parameter setting is needed based on mainnet such as this denom case of the upgrade handler in the PR or a separate PR

Reply:
Alright, I'll handle this in a separate PR.